zerver/models/streams.py

```diff
@@ -180,6 +181,7 @@ def is_history_public_to_subscribers(self) -> bool:
     # * "realm" and "recipient" are not exposed to clients via the API.
     API_FIELDS = [
         "date_created",
+        "creator_id",
```

Comment:
Put this field first; the values are clearly intended to be alphabetized.

Reply:
I have fixed this here and in other places as needed.